app/features/index.py

```diff
@@ -0,0 +1,51 @@
+'''
+    This file purpose is for developers to add their features to the database
+    in one convenient place, every time the system loads up it's adding and
+    updating the features in the features table in the database.
+
+    To update a feature, The developer needs to change the name or the route
+    and let the system load, but not change both at the same time otherwise
+    it will create junk and unnecessary duplicates.
+
+    *   IMPORTANT - To enable features panel functionlity the developer must  *
+    *   add the feature_access_filter decorator to ALL the feature routs      *
+    *   Please see the example below.                                         *
+
+    Enjoy and good luck :)
+'''
+
+'''
+Example to feature stracture:
+
+{
+    "name": '<feature name>',
+    "route": '/<the route like: /features >',
+    "description": '<description>',
+    "creator": '<creator name or nickname>'
```

Comment:
Is "creator" really necessary? Should developers who worked on non-features place their names on pieces of code they've added all over the site?

Reply:
It's like a an app on the Appstore you have the app some description and who created it.

There is some one who work on a credit page, so like you said developers who worked on non-features will have their name on the site.